index.html

```diff
@@ -234,6 +255,20 @@ <h1 id="timeDisplay">03:00</h1>
         }
       });
 
+      editTimeButton.addEventListener("click", () => {
+        if (!isTimerRunning) {
+          userTime = prompt(
+            "New Time formatted " + defaultTime/60 + ", "
```

Comment:
What do you think about making this just a static number, so the prompt reads something like "Possible formats: 8, 8:00, or 400s? And it gives some odd numbers occasionally:

![CleanShot 2023-12-11 at 16 26 13@2x](https://github.com/tydood/clocktowertimer/assets/153012987/041158d6-a193-42d2-8754-79e2a34878e7)

Is there a reason that I am missing why we couldn't make it static?

Reply:
nope, no real reason